.infra/Pulumi.adhoc.yaml

```diff
@@ -82,6 +82,5 @@ config:
     typeormUser: postgres
     tz: UTC
     urlPrefix: http://api.local.com
-    vordrIps: ""
```

Comment:
why removed?

Reply:
`process.env.VORDR_IPS?.split(',')` returns `['']`, which makes the `isInSubnet` subnet fail. So if I remove it, it falls back to `[]`

Reply:
add `filter(Boolean)`?

Reply:
Yeah, just did that now